geotrek/common/viewsets.py

```diff
@@ -6,9 +6,10 @@ class GeotrekMapentityViewSet(MapEntityViewSet):
     """ Custom MapentityViewSet for geotrek. """
 
     permission_classes = [permissions.DjangoModelPermissionsOrAnonReadOnly]
+    mapentity_list_class = None
```

Comment:
```suggestion
    mapentity_list_class = []
```
I suggest to keep same type as empty list by default